types/node/test/buffer.ts

```diff
@@ -423,7 +423,7 @@ buff.writeDoubleBE(123.123, 0);
 
 {
     // $ExpectType Blob | undefined
-    resolveObjectURL(URL.createObjectURL(new NodeBlob([''])));
+    resolveObjectURL(URL.createObjectURL(new Blob([''])));
```

Comment:
Why did this one change?

Reply:
This is a personal preference of mine, I don't like imports with names that shadow globals. (It can hide problems especially when the globals might be different as in this case.)  If there's any problem with it you're more than welcome to change it back.

Reply:
`Blob as NodeBlob` is imported in this file and the other references in the test file use `NodeBlob`, so I just wasn’t sure why this one specifically is changing to `Blob`?

Reply:
With `"dom"` included, `URL.createObjectUrl` comes from the DOM, and requires a DOM Blob. A node Blob does not work.